com.studio08.xbplay.yml

```diff
@@ -0,0 +1,63 @@
+app-id: com.studio08.xbplay
```

Comment:
`studio08.com` doesn't seem to be related, not do I see it mentionned anywhere. Is it really controlled by the upstream developers?

Reply:
I am the only developer of this app, there are no upstream developers?

Reply:
@hfiguiere  dang, I have `.net` not `.com`. Looks like I'll need to change literally everything? :(